### PR TITLE
Update libg

### PR DESCRIPTION
**Pull request by aparajit-pratap**
### Purpose

Update LibG for the following updates:
- Fix regression with `Curve.SplitByPoints` and `Curve.SplitByParameter` for PolyCurve
- 

### Declarations

Check these if you believe they are true

- [ ] The codebase is in a better state after this PR
- [ ] Is documented according to the [standards](https://github.com/DynamoDS/Dynamo/wiki/Coding-Standards)
- [ ] The level of testing this PR includes is appropriate
- [ ] User facing strings, if any, are extracted into `*.resx` files
- [ ] All tests pass using the self-service CI.
- [ ] Snapshot of UI changes, if any.
- [ ] Changes to the API follow [Semantic Versioning](https://github.com/DynamoDS/Dynamo/wiki/Dynamo-Versions) and are documented in the [API Changes](https://github.com/DynamoDS/Dynamo/wiki/API-Changes) document.
- [ ] This PR modifies some build requirements and the readme is updated

### Release Notes

 Fix regression with `Curve.SplitByPoints` and `Curve.SplitByParameter` for PolyCurve

### Reviewers

(FILL ME IN) Reviewer 1  (If possible, assign the Reviewer for the PR)

(FILL ME IN, optional) Any additional notes to reviewers or testers.

### FYIs

(FILL ME IN, Optional) Names of anyone else you wish to be notified of

### CODE REVIEW

File: src/DynamoCoreWpf/DynamoCoreWpf.csproj [hunk 1]
```diff
@@ -70,6 +70,8 @@
   </ItemGroup>
   <ItemGroup>
   <PackageReference Include="DynamoVisualProgramming.LibG_229_0_0" Version="2.17.1.3827"/>
+  <PackageReference Include="DynamoVisualProgramming.LibG_229_0_0" Version="2.17.1.3967" />
+  <PackageReference Include="AvalonEdit" Version="4.3.1.9430" CopyXML="true" />
```
Comment (mjkkirschner) [inline src/DynamoCoreWpf/DynamoCoreWpf.csproj:74]: whats this?

File: src/Libraries/CoreNodes/CoreNodes.csproj [hunk 2]
```diff
@@ -23,6 +23,14 @@
       <ExcludeAssets>runtime</ExcludeAssets>
     </PackageReference>
     <Reference Include="PresentationCore" />
+  </ItemGroup>
+  <ItemGroup Condition=" '$(TargetFramework)' == 'net6.0' ">
```
Comment (mjkkirschner) [inline src/Libraries/CoreNodes/CoreNodes.csproj:27]: huh?

**Comment by mjkkirschner** [comment]
I think there are some changes from master that should not be in this branch related to .net6

**Comment by aparajit-pratap** [comment]
Ugh, I cherry-picked the changes from master. Not sure how this happened, but I'll redo this. Thanks for catching it.

**Comment by aparajit-pratap** [comment]
@mjkkirschner this now looks just the same as the PR in master. Merging.

**Pull request merged**